### DISCUSSION TITLE
Feature Request: Support Databases with Replication Support

### DISCUSSION THREAD

**Discussion by VinayaSathyanarayana** [prisma/prisma#6424] [Q&A] [answered]
- [ ] Please provide support for databases with replication capability like [CouchDB](https://couchdb.apache.org/)
- [ ] Please provide documentation on how one can implement replication for other databases like ```postgresql```

**Comment by janpio** [comment, marked as answer]
Please open separate issues about these things if they do not exist yet (I could not find any quickly). 
We are happy to look into these separately.

**Comment by WilliamFalci** [comment]
![PRSIMA CLUSTER (3)](https://user-images.githubusercontent.com/36926081/173050782-ac6fb351-1cdc-4b78-8683-c9aa3c4542fd.png)

This is my own solution on use prisma in cluster with replication or without replication

[Prisma Cluster](https://github.com/WilliamFalci/Prisma-Cluster)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
